src/lib/hooks/react-use.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * This is an ES adapter for react-use CommonJS build.
+ *
+ * The problem with the react-use ESM build is that it's only working
+ * with a permissive interop import logic (typescript / babel),
+ * but fails when a named import done from within a node native ES module environment.
+ *
+ * This adapter re-exports individual hooks as ES-friendly named exports.
+ */
+
+import _useLatest from 'react-use/lib/useLatest.js';
```

Comment:
I recommend using `react-hookz` analog of this function instead of unmaintained `react-use` library.

https://react-hookz.github.io/web/?path=/docs/miscellaneous-usesyncedref--example